core/variant/variant.h

```diff
@@ -518,6 +518,7 @@ class Variant {
 	static String get_callable_error_text(const Callable &p_callable, const Variant **p_argptrs, int p_argcount, const Callable::CallError &ce);
 
 	//dynamic (includes Object)
```

Comment:
I guess the comment applied to `get_method_list` and `has_method`. Does it also apply to the new one? If not it's confusing now.